mlir/test/fusion/pr-e2e/mixr-slice-convolution-add-small.mlir

```diff
@@ -0,0 +1,9 @@
+// RUN: rocmlir-gen -fut mlir_slice_convolution_add --arch %arch --clone-harness %s | rocmlir-driver -kernel-pipeline=migraphx | rocmlir-driver -host-pipeline=migraphx,highlevel | rocmlir-gen -ph -rand 1 -rand_type float -fut mlir_slice_convolution_add_wrapper --verifier clone -relDiff_threshold 0.000002 - | rocmlir-driver -host-pipeline mhal -kernel-pipeline full | xmir-runner --shared-libs=%linalg_test_lib_dir/libmlir_rocm_runtime%shlibext,%conv_validation_wrapper_library_dir/libconv-validation-wrappers%shlibext,%linalg_test_lib_dir/libmlir_runner_utils%shlibext,%linalg_test_lib_dir/libmlir_float16_utils%shlibext,%linalg_test_lib_dir/libmlir_c_runner_utils%shlibext,%linalg_test_lib_dir/libmlir_async_runtime%shlibext --entry-point-result=void | FileCheck %s
+// ALLOW_RETRIES: 2
+// CHECK: [1 1 1]
+func.func private @mlir_slice_convolution_add(%arg0: tensor<1x2x4x4xf32>, %arg1: tensor<1x8x4x4xf32>, %arg2: tensor<2x2x1x7xf32>) -> tensor<1x2x4x4xf32> attributes {arch = "gfx908:sramecc+:xnack-", kernel = "mixr", num_cu = 120 : i64} {
```

Comment:
We probably want to take the `arch` off of this?

Reply:
done